modules/microshift-updating-rpms-ostree.adoc

```diff
@@ -0,0 +1,65 @@
+//Module included in the following assemblies:
+//
+//*  microshift_updating/microshift-update-rpms.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-updates-rpms-ostree_{context}"]
+= Applying updates on an OSTree system
+
+Tp update {product-title} on an `rpm-ostree` system such as {op-system-ostree-first}, embed the new version of {product-tile} on a new operating system image. The steps you use depends on how your existing deployment is set up. The following procedure outlines the general steps you can take, with links to the {op-system-ostree} documentation. The {op-system-ostree} documentation is your resource for specific details on building an updated operating system image. Back up and system rollback are automatic with this update type.
```

Comment:
```suggestion
To update {product-title} on an `rpm-ostree` system such as {op-system-ostree-first}, embed the new version of {product-tile} on a new operating system image. The steps you use depends on how your existing deployment is set up. The following procedure outlines the general steps you can take, with links to the {op-system-ostree} documentation. The {op-system-ostree} documentation is your resource for specific details on building an updated operating system image. Back up and system rollback are automatic with this update type.
```

Reply:
There is a typo in the second {product-title} instance in the statement above. It is being spelled {product-tile}